FitBitApi/FitBitApi/Modules/Presenter/Controller/PresenterViewController.swift

```diff
@@ -0,0 +1,45 @@
+//  
+//  PresenterViewController.swift
+//  FitBitApi
+//
+//  Created by Wishell on 14.11.2018.
+//  Copyright © 2018 Wishell. All rights reserved.
+//
+
+import UIKit
+
+final class PresenterViewController: UIViewController {
+    
+    var model: PresenterModelInput!
+    lazy var contentView: PresenterViewInput = { return view as! PresenterViewInput }()
+    let dataSource = DataSource()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        contentView.stopIndicator()
```

Comment:
stopIndicator?
we should use start - if it activity indicator, because next line starts model loading